yarn.lock

```diff
@@ -2674,6 +2695,49 @@ lcid@^1.0.0:
   dependencies:
     invert-kv "^1.0.0"
 
+lerna@^2.5.1:
+  version "2.5.1"
+  resolved "https://registry.yarnpkg.com/lerna/-/lerna-2.5.1.tgz#d07099bd3051ee799f98c753328bd69e96c6fab8"
```

Comment:
I don't think this kind of dependency will work, we need to be calling the locally built lerna.

Reply:
I thought we could track the second to latest version here. The issue with using the local version would be that there is nothing there to do the bootstraping then (if we e.g. built via babel)

Reply:
If we're going for node v8.x compatibility, we might as well convert stuff to CommonJS (require) and avoid the transpilation entirely. Nice and fast, but a lot more work.

Reply:
That still has the issue we have no lerna to do the root + leaf installs before said install, right?

Reply:
hrm, I'm not sure, with yarn workspaces in play. I feel like a local path as a dependency in the root package should be possible?
```json
"dependencies": {
  "lerna": "./packages/cli"
}
```

Reply:
Worth a try :)

Reply:
The current early CI failures are what I tried to avoid: https://travis-ci.org/lerna/lerna/jobs/306655406#L542